examples/Generator/inverse_trainer_xyz.py

```diff
@@ -72,6 +79,7 @@ def on_sample_start(self):
 
         # Iterate over samples in the batch
         for j in range(len(desc[0])):
+
```

Comment:
Why there is a single empty line, but it does not change anything.

Reply:
I don't know, probably I just thought it's more aesthetically pleasing.